text/1020-triggers.rst

```diff
@@ -0,0 +1,198 @@
+::
+
+    Status: Draft
+    Type: Feature
+    Created: 2022-12-15
+    Authors: Michel J. Sullivan <sully@msully.net>
+
+==================
+RFC 1020: Triggers
+==================
+
+This RFC proposes a mechanism for triggers.
+
+Some text stolen from an earlier RFC by Elvis.
+
+Motivation
+==========
+
+Triggers are useful for things such as generating audit logs and
+generating external events.
+
+Note
+====
+
+This version of the trigger RFC reflects a split between "triggers"
+and "mutation rewrites", which will come in another RFC.
+
+Specification
+=============
+
+Synopsis::
+
+    type <type-name> "{"
+        trigger <name>
+	    {after | after commit of}
```

Comment:
Add explicit "before" modifier? That's very useful for readability (and later for introspection).

Reply:
There's no before option